src/open_inwoner/openzaak/models.py

```diff
@@ -442,6 +442,14 @@ class ZaakTypeStatusTypeConfig(models.Model):
         verbose_name=_("Call to action text"),
         help_text=_("The text displayed on the call-to-action button"),
     )
+    case_link_text = models.CharField(
+        max_length=40,
+        default="Bekijk aanvraag",
```

Comment:
I think I'd prefer it if this field is optional (`blank=True`) and have the default `"Bekijk aanvraag"` in `status.py`. That way admins can simply make the `case_link_text` empty if they want to use the default again